build/toolchain/gcc_toolchain.gni

```diff
@@ -122,14 +122,20 @@ template("gcc_toolchain") {
       coverage_flags = "-fprofile-instr-generate -fcoverage-mapping"
     }
 
+    if (host_os == "mac") {
```

Comment:
`-i''` that is, an empty string with no space after the `-i` should work on both, but maybe it's better to be explicit like this.

Reply:
In my local testing on macOS `-i''` did not work.

It looked like it was being parsed as equivalent to `-i`, causing the next argument (the  sed command) to be used as the backup file extension.

Reply:
Ah, maybe it only works with `-e` before the command to delineate the empty string from the command like: `sed -i'' -e 's@/b/f/w@../..@g' $depfile`.

Reply:
I tried that originally but found that it was creating a backup file with the extension `-e` added to the filename

Reply:
Hahaha. Okay. I'm glad you checked =)